pkt-line.c

```diff
@@ -191,7 +191,7 @@ int packet_write_fmt_gently(int fd, const char *fmt, ...)
 
 static int packet_write_gently(const int fd_out, const char *buf, size_t size)
 {
-	static char packet_write_buffer[LARGE_PACKET_MAX];
+	char packet_write_buffer[LARGE_PACKET_MAX];
```

Comment:
I was in a `64 bit state of mind` at the time.  :-)
I want to make another pass on these type things before I submit it.

When I wrote this I was thinking I'd have multiple threads in the foreground process that each were managing a single conversation, so I could have a stack buffer for the duration of the write and not step on concurrent conversations in other threads.  And then maybe later move it to TLS.
  
But the parallel-checkout design evolved to where the main thread in the foreground process manages multiple conversations without any threads, so TLS wouldn't be helpful.

It might be actually better to move the buffer to hang off of some connection-instance data of some type, but that would touch a lot of code.

I'll revisit both this and the malloc question in a later cleanup round.

Reply:
Omitting all pkt-line changes from the next draft of the RFC.